src/main/java/domain/Lotto.java

```diff
@@ -0,0 +1,31 @@
+package domain;
+
+import java.util.Collections;
+import java.util.List;
+
+public class Lotto {
+
+    public static final int LOTTO_NUMBER_BOUND = 45;
```

Comment:
```suggestion
    private static final int LOTTO_NUMBER_BOUND = 45;
```
그럼 온전히 `Lotto`가 45에 대한 정보를 가질 수 있을 것 같아요!